client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx

```diff
@@ -117,9 +117,10 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
     updateIssueQuantity(allocatedQuantity);
   };
 
-  const handleIssueQuantityChange = (quantity: number) => {
+  const handleIssueQuantityChange = (quantity?: number) => {
     setIssueQuantity(quantity);
-    allocate(quantity, Number(packSizeController.selected?.value));
+    if (quantity !== undefined)
```

Comment:
This is the only one that really required any special handling for the `undefined` value, and it's not much.